### PR TITLE
Remove old repositories links

### PR DESCRIPTION
**Pull request by ipanimnim**
Redundant and confusing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,15 +15,14 @@ Inside of each category you'll find a **README.md** file, which will contain a m
 
 *Happy Integration!*
 
+## Note
+
+This (main) branch requires Spring Integration 5.0 or above. For samples running against earlier versions of Spring Integration, use the __SI4.3.x__ and other branches.
```
Comment (artembilan) [inline README.md:20]: This has to be up-to-date: `6.0` is the current one. `5.5.x` for the previous supported version.

**Review by artembilan — CHANGES_REQUESTED** [review]

**Comment by ipanimnim** [comment]
@artembilan I've updated the PR

File: README.md (hunk 1 above)
Comment (ipanimnim) [reply README.md]: Updated

**Comment by artembilan** [comment]
Sorry, it looks like I have missed your PR when had done this: https://github.com/spring-projects/spring-integration-samples/commit/67677e210f05565974f23b94b6722869adf81886.

So, this is closed in favor of that fix.

Thank you anyway!

**Pull request closed without merging**